.github/workflows/build_and_test.yml

```diff
@@ -11792,6 +11792,8 @@ jobs:
         make -j4
     - name: Build CMake Tests
       run: |
+        echo Show setenv.sh
+        cat OpenDDS/setenv.sh
```

Comment:
These lines were just for diagnosing. We can remove them since the build is successful now.